formatted_text.md

```diff
@@ -56,11 +56,9 @@ Allowed in inline contexts.
 * `<ul>...</ul>` - bullet list, list items are in `<li>...</li>`
 * `<ol>...</ol>` - numbered list, list items are in `<li>...</li>`
 * `<details><summary>Recipes</summary>...</details>` - a foldable content section (a "spoiler"). `summary` is optional - if omitted, it may default to something like "click here to open" (viewer-dependent). Note: `summary` must be the first child of `details` - there must not even be whitespace in between.
-* Whitespace - double-newline should be similar to HTML's `<p>` and single newline should be similar to HTML's `<br>`
+* `<br/>`, `<p/>` - as in HTML. Note that `<p>` is an empty element, as it represents a paragraph break, not a paragraph.
```

Comment:
From my understanding, `<br/>` and `<p/>` are the same. Why have two different names for the same thing?

Reply:
I'll try doing one in this comment: here's a br before  
after, and here's a p before

after

Reply:
Ah, I see, thanks. Resolved.